pexpect_serial/serial_spawn.py

```diff
@@ -21,7 +21,13 @@
 
 """
 
-from pexpect import spawn
+# pexpect.spawn is not supported on some systems (i.e. Windows(TM))
+# See https://pexpect.readthedocs.io/en/stable/overview.html#pexpect-on-windows for details.
+import os
+if os.name in ['nt']:
```

Comment:
AFAICT this condition is not even necessary. Using `pexpect.fdexpect.fdspawn` _should_ be the right choice on any operating system.

To keep it "as is" as much as possible though, I've added this guard here. Feel free to remove it.